packages/bus-expression-evaluator-aggregate/lib/ActorExpressionEvaluatorAggregate.ts

```diff
@@ -0,0 +1,47 @@
+import type { IAction, IActorArgs, IActorOutput, IActorTest, Mediate } from '@comunica/core';
+import { Actor } from '@comunica/core';
+import type { ExpressionEvaluatorFactory } from '@comunica/expression-evaluator';
+import type * as RDF from '@rdfjs/types';
+import type { Algebra } from 'sparqlalgebrajs';
+
+/**
+ * A comunica actor for expression-evaluator-aggregate events.
+ *
+ * Actor types:
+ * * Input:  IActionExpressionEvaluatorAggregate:      TODO: fill in.
+ * * Test:   <none>
+ * * Output: IActorExpressionEvaluatorAggregateOutput: TODO: fill in.
+ *
+ * @see IActionExpressionEvaluatorAggregate
+ * @see IActorExpressionEvaluatorAggregateOutput
+ */
+export abstract class ActorExpressionEvaluatorAggregate extends Actor<
```

Comment:
Perhaps `ActorBindingsAggregatorFactory` or `ActorExpressionAggregatorFactory` might be a more convenient name.

Reply:
ActorBindingsAggregatorFactory :)